src/FixedDataTableCell.react.js

```diff
@@ -121,13 +121,18 @@ var FixedDataTableCell = createReactClass({
     var columnResizerComponent;
     if (props.onColumnResize) {
       var columnResizerStyle = {
-        height
+        height,
+        opacity: props.isHoveringResizerKnob ? 1 : 0,
       };
+      console.log('isHovering: ', props.isHoveringResizerKnob)
```

Comment:
@swifttarrow looks like you forgot to remove these logs. 🤔 Do we have the linter turned off for this file extension? I'd expect it'd have caught this

Reply:
@pete-bc Hrm, that's odd. Lemme fix; thanks for catching!